README.md

```diff
@@ -138,30 +153,29 @@ Congratulations! You have successfully started the flow. Carry on reading the ne
 - Handle callbacks
 - Remove the SDK from the page
 - Customise the SDK
-- Upload files to the Onfido API
 - Create checks
 
 ## Handling callbacks
 
-A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback, you would typically send the captured images to your backend server, where you would then [upload files](#uploading-files-to-onfido-api) and [create checks](#creating-checks) using the [Onfido API](https://documentation.onfido.com/).
+A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback you would typically trigger your backend to create a check using the [Onfido API](https://documentation.onfido.com/).
 
 - **`onReady {Function} optional`**
```

Comment:
we should remove this callback though, since it's no longer working. see https://github.com/onfido/onfido-sdk-ui/issues/153